extras/tmux/evergarden_tmux.conf

```diff
@@ -0,0 +1,53 @@
+set -ogq @evergarden_style "hard"
```

Comment:
this should default to medium. i know mocha is the default for catppuccin (and that seems to be what you have this based on) but evergarden just uses medium as the default.

Reply:
Default with medium if the user not specifically config it.